glib/example/sqlite.c

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdlib.h>
+
+#include <adbc-glib/adbc-glib.h>
+#include <arrow-glib/arrow-glib.h>
+
+int main(int argc, char** argv) {
+  GADBCDatabase* database;
+  GADBCConnection* conn;
+  GError* error = NULL;
+
+  database = gadbc_database_new(&error);
+
+  if (error != NULL) {
```

Comment:
Could you use `(error)` instead of `(error != NULL)`? We use this style in our existing code.

```suggestion
  if (error) {
```

Reply:
For this case, could you check `database` instead of `error`?
We want users to check `database`.

```suggestion
  if (!database) {
```